test/e2e_node/topology_manager_test.go

```diff
@@ -666,16 +668,21 @@ func setupSRIOVConfigOrFail(f *framework.Framework, configMap *v1.ConfigMap) *sr
 	dpPod, err := f.ClientSet.CoreV1().Pods(metav1.NamespaceSystem).Create(context.TODO(), dp, metav1.CreateOptions{})
 	framework.ExpectNoError(err)
 
+	if err = e2epod.WaitForPodCondition(f.ClientSet, metav1.NamespaceSystem, dp.Name, "Ready", 120*time.Second, testutils.PodRunningReady); err != nil {
+		framework.Logf("SRIOV Pod %v took too long to enter running/ready: %v", dp.Name, err)
+	}
+	framework.ExpectNoError(err)
+
 	sriovResourceName := ""
 	var sriovResourceAmount int64
 	ginkgo.By("Waiting for devices to become available on the local node")
 	gomega.Eventually(func() bool {
 		node := getLocalNode(f)
-		framework.Logf("Node status: %v", node.Status.Capacity)
+		framework.Logf("Node status allocatable: %v", node.Status.Allocatable)
```

Comment:
this log should be moved into findSRIOVResource for the sake of clarity - we should access `node.Status.Allocatable` only in the callee, not both in the caller and the callee.

Reply:
Done.